site/content/usage/experimental/01-gitops.md

```diff
@@ -32,57 +32,82 @@ Or use a config file:
+[ℹ]  Generating public key infrastructure for the Helm Operator and Tiller
+[ℹ]    this may take up to a minute, please be patient
+[!]  Public key infrastructure files were written into directory "/var/folders/zt/sh1tk7ts24sc6dybr5z9qtfh0000gn/T/eksctl-helm-pki330304977"
+[!]  please move the files into a safe place or delete them
+[ℹ]  Generating manifests
+[ℹ]  Cloning git@github.com:weaveworks/cluster-1-gitops.git 
+Cloning into '/var/folders/zt/sh1tk7ts24sc6dybr5z9qtfh0000gn/T/eksctl-install-flux-clone-142184188'...
+remote: Enumerating objects: 74, done.
+remote: Counting objects: 100% (74/74), done.
+remote: Compressing objects: 100% (55/55), done.
+remote: Total 74 (delta 19), reused 69 (delta 17), pack-reused 0
+Receiving objects: 100% (74/74), 30.57 KiB | 381.00 KiB/s, done.
+Resolving deltas: 100% (19/19), done.
 [ℹ]  Writing Flux manifests
-[ℹ]  Installing Flux into the cluster
+[ℹ]  Applying manifests
 [ℹ]  created "Namespace/flux"
-[ℹ]  created "flux:ServiceAccount/flux"
-[ℹ]  replaced "ClusterRole.rbac.authorization.k8s.io/flux"
-[ℹ]  replaced "ClusterRoleBinding.rbac.authorization.k8s.io/flux"
-[ℹ]  created "flux:Deployment.apps/flux"
 [ℹ]  created "flux:Secret/flux-git-deploy"
 [ℹ]  created "flux:Deployment.apps/memcached"
+[ℹ]  created "flux:ServiceAccount/flux"
+[ℹ]  created "ClusterRole.rbac.authorization.k8s.io/flux"
+[ℹ]  created "ClusterRoleBinding.rbac.authorization.k8s.io/flux"
+[ℹ]  created "flux:ConfigMap/flux-helm-tls-ca-config"
+[ℹ]  created "flux:Deployment.extensions/tiller-deploy"
+[ℹ]  created "flux:Service/tiller-deploy"
+[ℹ]  created "CustomResourceDefinition.apiextensions.k8s.io/helmreleases.helm.fluxcd.io"
+[ℹ]  created "flux:ServiceAccount/tiller"
+[ℹ]  created "ClusterRoleBinding.rbac.authorization.k8s.io/tiller"
+[ℹ]  created "flux:ServiceAccount/helm"
+[ℹ]  created "flux:Role.rbac.authorization.k8s.io/tiller-user"
+[ℹ]  created "kube-system:RoleBinding.rbac.authorization.k8s.io/tiller-user-binding"
+[ℹ]  created "flux:Deployment.apps/flux"
 [ℹ]  created "flux:Service/memcached"
+[ℹ]  created "flux:Deployment.apps/flux-helm-operator"
+[ℹ]  created "flux:ServiceAccount/flux-helm-operator"
+[ℹ]  created "ClusterRole.rbac.authorization.k8s.io/flux-helm-operator"
+[ℹ]  created "ClusterRoleBinding.rbac.authorization.k8s.io/flux-helm-operator"
+[ℹ]  Applying Helm TLS Secret(s)
+[ℹ]  created "flux:Secret/flux-helm-tls-cert"
+[ℹ]  created "flux:Secret/tiller-secret"
+[!]  Note: certificate secrets aren't added to the Git repository for security reasons
+[ℹ]  Waiting for Helm Operator to start
+ERROR: logging before flag.Parse: E0820 16:05:12.218007   98823 portforward.go:331] an error occurred forwarding 60356 -> 3030: error forwarding port 3030 to pod b1a872e7e6a7f86567488d66c1a880fcfa26179143115b102041e0ee77fe6f9e, uid : exit status 1: 2019/08/20 14:05:12 socat[2873] E connect(5, AF=2 127.0.0.1:3030, 16): Connection refused
```

Comment:
Are we supposed to log this error?

Reply:
We can't help it, see #1067

Reply:
I can remove it from the listing, if you prefer that